attestation_statement_apple.go

```diff
@@ -0,0 +1,76 @@
+package webauthn
+
+import (
+	"crypto"
+	"crypto/sha256"
+	"crypto/subtle"
+	"fmt"
+
+	"github.com/pomerium/webauthn/cose"
+)
+
+// VerifyAppleAttestationStatement verifies that an AttestationObject's attestation statement is valid according to the
+// packed verification procedure.
+func VerifyAppleAttestationStatement(
+	attestationObject *AttestationObject,
+	clientDataJSONHash ClientDataJSONHash,
+) error {
+	certificate, err := attestationObject.Statement.UnmarshalCertificate()
```

Comment:
Should we cover the status of step (1) here?

> Verify that attStmt is valid CBOR conforming to the syntax defined above and perform CBOR decoding on it to extract the contained fields.

Reply:
We do CBOR decoding elsewhere